.eslintrc.json

```diff
@@ -15,6 +15,12 @@
   ],
   "ignorePatterns": [
     "rollup.config.js",
+    "src/contracts/*",
+    "src/neynarAPI/neynarV1API/openapi/*",
+    "src/neynarAPI/neynarV2API/openapi/*",
```

Comment:
- If you move the generated code to a `generated` folder in each of these directories then you'll automatically get them ignored by line 24

Reply:
NVM I see it has already been moved! You should be able to just delete these lines then

Reply:
👍